src/ArmTemplates/Common/Constants/GlobalConstants.cs

```diff
@@ -11,7 +11,7 @@ public static class GlobalConstants
         public const string ExtractName = "extract";
         public const string ExtractDescription = "Extract an existing API Management instance";
 
-        public const string ApiVersion = "2021-01-01-preview";
+        public const string ApiVersion = "2021-08-01";
         public const string LinkedAPIVersion = "2018-05-01";
```

Comment:
Should we update the LinkedAPIVersion too?  My suggestion is yes.

Reply:
Ye, we have to. It is used in `masterTemplate` generation. Thanks for that catch!